scripts/india_nhm/data/districts-local-directory.csv

```diff
@@ -0,0 +1,739 @@
+rowid,S.No.,StateCode,StateName(InEnglish),DistrictCode,DistrictVersion,DistrictName(InEnglish),DistrictName(InLocal),Census2001Code,Census2011Code,wikidataId
```

Comment:
Is this a copy of the one in covid19_india too?  Maybe we can place it in a common location, like "scripts/india_geos/..."

Reply:
I used a direct url import from LGD portal. Instead using local csv file.